applications/nrf_desktop/README.rst

```diff
@@ -1556,6 +1556,8 @@ The B0 bootloader requires the following options enabled:
 * :kconfig:option:`CONFIG_SB_SIGNING_KEY_FILE` - Required for providing the signature used for image signing and verification.
 * :kconfig:option:`CONFIG_FW_INFO` - Required for the application versioning information.
 * :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` - Enable this option to set the version of the application after you enabled :kconfig:option:`CONFIG_FW_INFO`.
+  For the DFU package generation, you need to update this configuration in the main application image and the ``s1_image`` child image.
+  You can do that by rebuilding the application from scratch or by changing the configuration independently in both images with the help of menuconfig.
```

Comment:
with the help of menuconfig ->in menuconfig / through menuconfig

Sorry for leaving comments instead, suggesting mode doesn't work for me for some reason.